ofcourse/yamls/site.yaml

```diff
@@ -7,6 +7,8 @@ course:
     times: Tues & Thurs 6:30PM - 7:45PM
     course: 4085.582.01
     location: Orange Hall (ORN)-1380
+    repo: https://github.com/decause/hfoss
+    prod: https://hfoss-ritigm.rhcloud.com
```

Comment:
How do you think we should set these defaults? This seems like the `[prod]` key could be gotten from the openshift env instead of hard coding.